src/utils/gaa.js

```diff
@@ -2260,4 +2263,12 @@ export class GaaMetering {
       }
     });
   }
+
+  static getSubscriptionTimestamp() {
+    if (GaaMetering.userState && GaaMetering.userState.subscriptionTimestamp) {
```

Comment:
Could this method body be simplified to `return GaaMetering?.userState?.subscriptionTimestamp` ?

Reply:
Sounds good, thanks!